docs/app/pages/charts/charts-sections/sankey-chart/sankey-chart/sankey-chart.component.html

```diff
@@ -1,5 +1,6 @@
 <ux-sankey-chart
     class="sankey-chart"
+    [class.theme-micro-focus-2020]="theme === SiteThemeId.MicroFocus2020"
```

Comment:
This class doesn't seem to be getting applied consistently whenever switching between themes. It's a bit tricky to explain what I mean but if you switch between the themes then you should the font weight being changed whenever you hover over a node.

1. Start on the default theme (after refreshing) and hover over one of the nodes
1. Switch to 2020 and hover over one of the nodes again
1. Switch back to the default theme and when you hover again you see this:

[![Image from Gyazo](https://i.gyazo.com/01e47f9ad3d3808c052b14b5382b45f5.gif)](https://gyazo.com/01e47f9ad3d3808c052b14b5382b45f5)

Reply:
Thanks, good catch on this, I guess it's because the template is a child of the component so the change detection doesn't work. I've backed it out because I haven't found a trivial fix and it doesn't make much difference to the example.